src/test/java/com/crud/tasks/controller/TaskControllerTest.java

```diff
@@ -46,34 +50,98 @@ public void shouldFetchTasksList() throws Exception {
 
         //Given
         List<TaskDto> listTasksDto = new ArrayList<>();
-        listTasksDto.add(new TaskDto(1L, "Test Task", "Test content"));
+        listTasksDto.add(new TaskDto(1L, "Test", "Test content"));
 
         when(taskMapper.mapToTaskDtoList(service.getAllTasks())).thenReturn(listTasksDto);
 
         //When & Then
-        mockMvc.perform(get("/v1/task")
+        mockMvc.perform(get("/v1/task/getTasks")
                 .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().is(200))
                 .andExpect(jsonPath("$", hasSize(1)))
                 .andExpect(jsonPath("$[0].id", is(1)))
-                .andExpect(jsonPath("$[0].title", is("Test Task")))
-                .andExpect(jsonPath("$[0].content", is("Test Task content")));
+                .andExpect(jsonPath("$[0].title", is("Test")))
+                .andExpect(jsonPath("$[0].content", is("Test content")));
     }
 
     @Test
-    public void shouldFetchTask() {
+    public void shouldFetchTask() throws Exception {
+
+        //Given
+        Task task = new Task(1L, "Test", "Test content");
+        TaskDto taskDto = new TaskDto(1L, "Test", "Test content");
+
+        when(taskMapper.mapToTaskDto(task)).thenReturn(taskDto);
+        when(service.getTask(1L)).thenReturn(Optional.of(task));
+
+        //When & Then
+        mockMvc.perform(get("/v1/task/getTask?taskId=1")
+                .contentType(MediaType.APPLICATION_JSON)
+                .param("taskId", "1"))
```

Comment:
Duplikacja - jeśli podajesz parametr `taskId` przez `.param(...)`, to nie musisz go umieszczać w URL-u w metodzie `get(...)`.